pilot/pkg/bootstrap/certcontroller.go

```diff
@@ -229,8 +233,9 @@ func (s *Server) updateRootCertAndGenKeyCert() error {
 	return nil
 }
 
-// initCertificateWatches sets up watches for the plugin dns certs.
-func (s *Server) initCertificateWatches(tlsOptions TLSOptions) error {
+// initFileCertificateWatches sets up watches for the plugin dns certs
+// when /var/run/secrets/istiod/tls is used. Not to be confused with the /etc/cacerts path.
```

Comment:
Same as above this is not quite right, the patch can be different

Reply:
'or custom location' ? 

Working on a doc with all the paths and options - the problem is we have too many options and too little explanation of what is where and in which order.